tests/integration/utils.py

```diff
@@ -364,7 +365,8 @@ def fetch_deps_and_check_output(
     repo.git.clean("-ffdx")
     repo.git.checkout(test_params.branch)
 
-    output_dir = tmp_path.joinpath(DEFAULT_OUTPUT)
+    dirname = fetch_output_dirname or DEFAULT_OUTPUT
```

Comment:
PEP8 purism: Since it defaults to `None` you better be explicit with a check for `None`. If, OTOH, you don't need it to be `None` then you should use a different default value (`""` will do in this case).

Reply:
Fixed.